pombola/kenya/static/js/survey.js

```diff
@@ -0,0 +1,75 @@
+(function() {
+
+    // Percentage of users to display the survey for
+    var p = 0.6;
+
+    // Is cookie existing?
+
+    var ct = null;
+    var ca = document.cookie.split(';');
+    for(var i=0;i < ca.length;i++) {
+        var c = ca[i];
+        while (c.charAt(0)==' ') {
```

Comment:
Is there a particular reason to strip leading whitespace like this, rather than, say, `c.replace(/^\s+/, '')` ?